app/controllers/admin/schedules_controller.rb

```diff
@@ -4,6 +4,7 @@
 module Admin
   class SchedulesController < ResourceController
     before_filter :check_editable_order_cycle_ids, only: [:create, :update]
+    before_filter :check_dependent_subscriptions, only: [:destroy]
```

Comment:
In the case of a single action, as in here, I think a filter doesn't bring any benefit. The indirection actually makes things a litlle bit harder to read.

Reply:
Hey @sauloperez, I know you hate single-use before filters 😉, but in the case where you want to return early they actually have a special property. When you `render` or `redirect_to` from a before filter, the action is never hit. So it is useful for rendering an error when a certain condition is not met, for example. This is *possible* but more difficult to achieve using a simple method call from within the action: you need to a) add and extra render to the action method, b) return from the filter method but return a value which the action method understands, or c) check `performed?`. All of these option add unnecessary clutter the action method IMHO.

Reply:
:+1: